src/Plugin.php

```diff
@@ -140,18 +140,29 @@ private function loadInstalledPaths()
      */
     private function saveInstalledPaths()
     {
-        // By default we delete the installed paths
-        $arguments = ['--config-delete', 'installed_paths'];
+        $configKey = 'installed_paths';
 
-        // This changes in case we do have installed_paths
+        // Check if we found installed_paths to set.
         if (count($this->installedPaths) !== 0) {
-            $arguments = ['--config-set', 'installed_paths', implode(',', $this->installedPaths)];
+            $paths = implode(',', $this->installedPaths);
+            $arguments = ['--config-set', $configKey, $paths];
+            $this->io->write(sprintf('PHP CodeSniffer Config <info>%s</info> <comment>set to</comment> <info>%s</info>', $configKey, $paths));
```

Comment:
As a matter of cleanup, I would suggest assigning the message to a separate variable (instead of calling `$this->io->write()`) and placing the call to io->write outside of the if/else.